nsfc-plugin/src/main/java/org/osc/controller/nsfc/api/NeutronSfcSdnControllerApi.java

```diff
@@ -58,28 +46,11 @@
                  SUPPORT_NEUTRON_SFC + ":Boolean=true"})
 public class NeutronSfcSdnControllerApi implements SdnControllerApi {
 
-    @Reference(target = "(osgi.local.enabled=true)")
-    private TransactionControl txControl;
-
-    @Reference(target = "(osgi.unit.name=nsfc-mgr)")
-    private EntityManagerFactoryBuilder builder;
-
-    @Reference(target = "(osgi.jdbc.driver.class=org.h2.Driver)")
-    private DataSourceFactory jdbcFactory;
-
-    @Reference(target = "(osgi.local.enabled=true)")
-    private JPAEntityManagerProviderFactory resourceFactory;
-
     private static final Logger LOG = LoggerFactory.getLogger(NeutronSfcSdnControllerApi.class);
 
     private static final String VERSION = "0.1";
```

Comment:
I guess this is the version for the plugin. we should enhance the version number, as this is a major enhancement.

Reply:
This one has not been changed for a while. I am not finding any references. It's the same in the NSC plugin